feature-tests/utils/invoke-apigateway-lambda.ts

```diff
@@ -0,0 +1,13 @@
+import { InvokeCommand, LambdaClient } from '@aws-sdk/client-lambda';
+
+export async function invokeApiGateWayLambda(testUserId: string, historyValue: boolean) {
+  const client = new LambdaClient({});
+  const command = new InvokeCommand({
+    FunctionName: 'ais-main-kg-InvokePrivateAPIGatewayFunction',
```

Comment:
are we sure this should be hardcoded?

Reply:
Thanks, we will resolve this in a future PR change https://govukverify.atlassian.net/browse/ATB-1209